wiki/Shortcut_Key_Reference/en.md

```diff
@@ -1,179 +1,208 @@
+They are listed out with their functions below.
+
+**Note:** some of these keys can be changed in the [Options](/wiki/Options/)!
+However, the list below presents the default keys.
+
+## Game-Wide
+
+These shortcuts work anywhere:
+
+- `Arrow Down`: Decrease volume.
+- `Arrow Up`: Increase volume.
+- `Alt` + `F4`: If playing/editing a beatmap, exit to song select; otherwise, this will close the osu!client.
+- `Ctrl` + `Alt` + `Shift` + `S`: Reload current skin (If playing a beatmap, this will queue the command to be executed after returning to song selection)
+- `Ctrl` + `F11`: Toggle frame times
+- `Ctrl` + `O`: Open options menu (does not work while playing in-game)
+- `F7`: Toggle frame limiter
+- `F8`: Toggle chat window
+- `F9`: Toggle extended chat window
+- `F10`: Toggle all mouse buttons (except for mouse wheel)
+- `F12`: Take a screenshot (hold `Shift` to upload instantly to the server for sharing.)
+- `Insert`: Minimize the osu!client into the notification area.
+- `Shift` + `F1`: Open [osu!status](https://twitter.com/osustatus "osu!status on twitter") (when connection error appears)
+
+## Main Menu
+
+- `Alt` + `Enter`: Toggle fullscreen mode
+- `Esc`: Go back one menu
+- `P`: Play mode
+- `P`, then `P`: Solo
+- `P`, then `M`: [Multi](/wiki/Multi)
+- `E`, or `P`, then `E`: Edit mode
+- `Z` or `F1`: Previous song (jukebox)
+- `X` or `F2`: Play from start (jukebox)
+- `C`: Toggle pause (jukebox)
+- `F4`: Stop playing and reset duration to 0 (jukebox)
+- `V` or `F5`: Next song (jukebox)
+- `J` or `F6`: Jump To (jukebox)
+- `R`: Random song (jukebox)
+- `D`: osu!direct ([osu!supporters](/wiki/osu!supporter) only)
+- `O`: Options
+- `Q` or `Esc`: Display exit dialog
+
+## Song Select
+
+- `F1`: Display the [mod](/wiki/Game_Modifiers) selection screen
+- `F2`: Randomly select a map
+- `Shift` + `F2`: Undo random map selection
+- `F3`: Display beatmap options
+- `F5`: Refresh listing (repair mode)
+- `Shift` + `Delete`: Delete current map
+- `Ctrl` + `1`: Switch to [osu!](/wiki/osu!)
+- `Ctrl` + `2`: Switch to [osu!taiko](/wiki/osu!taiko)
+- `Ctrl` + `3`: Switch to [osu!catch](/wiki/osu!catch)
+- `Ctrl` + `4`: Switch to [osu!mania](/wiki/osu!mania)
+- `Ctrl` + `Enter`: Play current map with [Auto](/wiki/Auto) mod
+- `Ctrl` + `Shift` + `Enter`: Play current map with [Cinema](/wiki/Cinema) mod
+- `Ctrl` + `A`: Toggle Auto mod
+- `Ctrl` + `F3` or `Ctrl` + `+`: Increase osu!mania speed (only in osu!mania mode)
+- `Ctrl` + `F4` or `Ctrl` + `-`: Decrease osu!mania speed (only in osu!mania mode)
+- `Ctrl` + `Backspace`: Delete last word (when using search)
+- `Up` or `Down`: Change selection
+- `Left` or `Right`: Change beatmap set
+- `Page Up` or `Page Down`: Change selection (large)
+- `Enter`: Make selection
+- `Shift` + `Left Arrow` or `Shift` + `Right Arrow`: Change group
+- `Shift` + `Enter`: Open/close group
+
+## Mods screen
+
+**Notes:**
+
+- The key references below assumes that none of the mods are selected.
```

Comment:
it looks cleaner imo (and because syntax highlighting on my editor breaks without it